### ISSUE TITLE
Spoiler Log Without Generating ROM

### ISSUE THREAD

**Issue by Nicholas-Steel** [Dessyreqt/alttprandomizer#200]
I for the life of me can not find the red Cane of Somaria and this is preventing me from accessing the 7th dungeon. I've gotten everything except the Cane, Mirror Shield and Red Armour. My seed is: C4067827

I figured the spoiler button in the randomizer program would let me see the locations of items but alas that does NOT seem to be its purpose, that button is for creating a brand new Seed with the locations for that new Seed revealed and will not let me input a seed of my own (it overwrites the seed I input in to the Seed box with a new randomly chosen Seed when I click the Random Spoiler button).

So uh, I would like to request a feature where I can generate a randomized ROM and later on, if I get stuck, be able to somehow then at that point in time generate a list of item locations (instead of only being able to at the time of creating the randomized ROM).

Basically a way to input a Seed, click Spoiler and see the item locations for that Seed.

**Comment by jakj** [comment]
Just put in the same seed (the thing that starts with C) in the randomizer program to re-generate the same rom, and click Create Spoiler Log this time. The seed should be in the file name and should show in the emulator when you start the rom.

**Comment by Nicholas-Steel** [comment]
Huh, so that's how you do it. Thanks.

**Comment by Nicholas-Steel** [comment]
Perhaps add a way to do this without having to generate another copy of the same ROM file in the process (Like what the Random Spoiler button does, but don't overwrite the Seed that the user may have inputted).

**Comment by Dessyreqt** [comment]
V7 will include the ability to view a spoiler with the seed and without having to generate a ROM.

**Comment by Nicholas-Steel** [comment]
Thanks.

**Comment by Dessyreqt** [comment]
This is in the master branch now.

**Comment by Nicholas-Steel** [comment]
Neat.
